package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "electron",
+  "version": "v0.35.0",
```

Comment:
I think I should probably add something to `bump-version.py` instead. Thanks for the info by the way.